tests/tests.rs

```diff
@@ -0,0 +1,109 @@
+extern crate froggy;
+
+use froggy::Storage;
+
+#[test]
+fn change_by_pointer() {
```

Comment:
the test is too big and has out of scope functions
let's test iteration separately from pinning  and writing